content/en/contribute/code/workflow.md

```diff
@@ -162,7 +162,7 @@ Issues in this column have passed AT and can be merged as soon as possible. The
 
 ### Done
 
-Issues in this column have passed acceptance testing and been merged into `master` and/or release branches ready for release.
+Issues in this column have passed acceptance testing and been merged into `main` and/or release branches ready for release.
```

Comment:
This whole file refers to code development work in other repos (eg: cht-core) which haven't yet switched to `main`. Do we expect that to be migrated soon (in which case this is fine), or is it worth holding off on changing this file until all repos are migrated?

Reply:
Not soon, as cht-core is the last repo that will be modified. I'll undo the changes on this file.

Reply:
I rolled back the changes for the suggested files.